modules/spark/build.sbt

```diff
@@ -0,0 +1,17 @@
+import Dependencies.Version._
+
+name := "pureconfig-spark"
+
+crossScalaVersions := Seq(scala212) //Spark does not support Scala 2.13 yet
```

Comment:
Getting CI errors from the exclusion of `scala213` here.

Reply:
https://github.com/pureconfig/pureconfig/pull/1095/checks?check_run_id=3041795063

Reply:
@ruippeixotog could you also have a look a this issue? It seems like if `scala213` is removed from any module it would fail.